ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -563,7 +588,10 @@ export default class ConfirmApproveContent extends Component {
         >
           {this.renderTitle()}
         </div>
-        <div className="confirm-approve-content__description">
+        <div
+          className="confirm-approve-content__description"
+          data-testid="confirm-approve-description"
+        >
```

Comment:
We can also avoid adding data-testid here if we use snapshot test.

Reply:
@jpuri Done [here](https://github.com/MetaMask/metamask-extension/pull/17992/commits/d7e12d4155ab6ed63a4b87f71350d9d801457c99#diff-49a399717cd27ce0bb0c752633bd4e684dd9ae6469f26f0a1dcaf607ffdc6d76).